samples/cpp/benchmark_app/property_map.hpp

```diff
@@ -0,0 +1,172 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <openvino/runtime/intel_gna/properties.hpp>
+#include <openvino/runtime/intel_gpu/properties.hpp>
+#include <openvino/runtime/intel_gpu/remote_properties.hpp>
+#include <openvino/runtime/intel_myriad/hddl_properties.hpp>
+#include <openvino/runtime/intel_myriad/myriad_properties.hpp>
+
+#define VALUE_2_STRING(name) #name
+#define INSERT_NAME_VALUE_MAP(out_map, value)        \
+    {                                                \
+        std::string str_key = VALUE_2_STRING(value); \
+        std::stringstream str_value;                 \
+        str_value << value;                          \
+        out_map[str_key] = str_value.str();          \
+    }
+
+enum PropertyType {
+    PROPERTY_TYPE_STRING = 0,
+    PROPERTY_TYPE_INTER,
+    PROPERTY_TYPE_UNSIGNED_INTER,
+    PROPERTY_TYPE_INTER64,
```

Comment:
Either INTEGER or INT. Inter does fit here.

Reply:
Ok, I will change it to INT.